egs/fame/v2/steps

```diff
@@ -0,0 +1 @@
+../v1/steps
```

Comment:
Probably this should point to the original steps directory (I think in WSJ?)

Reply:
You are right, it is a chain of links now, i.e. v2/steps --> ../v1/steps --> ../../sre08/v1/steps --> ../../wsj/s5/steps
I can fixed it if needed..